ceph_deploy/util/system.py

```diff
@@ -30,6 +31,52 @@ def is_systemd(conn):
         '/proc/1/comm'
     )
 
+def systemd_defaults_clustername(conn, cluster_name):
+    if not is_systemd:
+        return
+    sysconf_file_path = '/etc/sysconfig/ceph'
+    sysconf_dir_path = os.path.dirname(sysconf_file_path)
+    system_config_dir_exists = conn.remote_module.path_exists(sysconf_dir_path)
+    if not system_config_dir_exists:
+        conn.remote_module.safe_mkdir(sysconf_dir_path)
+    system_config_exists = conn.remote_module.path_exists(sysconf_file_path)
+    replacement = "CLUSTER=%s" % (cluster_name)
+    find_cluster = "^CLUSTER=.*$"
+    pattern = re.compile(find_cluster)
+    output_lines = []
+    has_replaced = False
+    if system_config_exists:
+        content = conn.remote_module.get_file('/etc/sysconfig/ceph')
+        for line in content.split('\n'):
+            stripped_line = line.strip()
+            match_details = pattern.match(line)
+            if match_details == None:
+                output_lines.append(stripped_line)
+                continue
+            match_content = line[match_details.start():match_details.end()]
+            if match_content == replacement:
+                if not has_replaced:
+                    output_lines.append(replacement)
+                    has_replaced = True
+                else:
+                    output_lines.append("#%s" % (stripped_line))
+                continue
+            if has_replaced == True:
+                output_lines.append("#%s" % (stripped_line))
+                continue
+            has_replaced = True
+            output_lines.append("#%s" % (stripped_line))
+            output_lines.append(replacement)
+
+    if has_replaced == False:
+        output_lines.append(replacement)
+    for index in range(len(output_lines)-1 , 0,-1):
+        if len(output_lines[index]) == 0:
+            del(output_lines[index])
+        else:
+            break
+    conn.remote_module.write_file(sysconf_file_path, "\n".join(output_lines) + '\n')
+
```

Comment:
Mind providing some unit tests for this new `systemd_defaults_clustername` function?